src/components/KMultiselect/KMultiselectItem.vue

```diff
@@ -16,11 +16,30 @@
         @click="handleClick"
       >
         <span class="k-multiselect-item-label mr-2">
-          <slot name="content">{{ item.label }}</slot>
+          <slot name="content">{{ item.label }}
+            <div
+              v-if="(item.selected && item.disabled)"
```

Comment:
nit: parenthesis aren't needed
```suggestion
              v-if="item.selected && item.disabled"
```